File: smallpict-java/src/main/java/com/smallpict/models/ImageFormat.java
package com.smallpict.models;

import com.fasterxml.jackson.annotation.JsonValue;

public enum ImageFormat {
    AUTO("auto"),
    AVIF("avif"),
    WEBP("webp"),
    JPEG("jpeg"),
    PNG("png");

    private final String value;

    ImageFormat(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return value;
    }

    @Override
    public String toString() {
        return value;
    }
}

File: smallpict-php/src/Models/ImageFormat.php
<?php

declare(strict_types=1);

namespace SmallPict\Models;

final class ImageFormat
{
    public const AUTO = 'auto';
    public const AVIF = 'avif';
    public const WEBP = 'webp';
    public const JPEG = 'jpeg';
    public const PNG = 'png';

    private function __construct()
    {
    }
}
